libs/s25main/mapGenerator/Random.cpp

```diff
@@ -36,6 +36,13 @@
 #include "mapGenerator/elevation/HeightMap.h"
 #include "mapGenerator/elevation/Level.h"
 
+namespace rttr {
+namespace mapGenerator {
+
+using namespace objects;
```

Comment:
Looks good so far. 👍 
It seems you are doing that already but just to make it explicit: `using namespace` is disallowed in headers (as it affects more than it should) and should be avoided generally. Sometimes it avoids repetitive mentioning of namespaces but often it helps making it clear what you are doing. E.g. `resources::check_placement(...)` is better than a `using namespace resources; check_placement(...)`.

Reply:
** Fixed**
There was no need for all the subfolders and therefore nested namespaces. I moved functions which fit together into a single file in mapGenerator folder and `rttr::mapGenerator` namespace.